src/cartservice/ddl/CartItems.ddl

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE CartItems (
+  userId STRING(1024),
+  productId STRING(1024),
+  quantity INT64,
+) PRIMARY KEY (userId, productId);
+
+CREATE INDEX CartItemsByUserId ON CartItems(userId);
```

Comment:
**Thought:**
This might be a waste of time, so let's ignore for now, but...
It looks like [cloud.google.com samples](https://cloud.google.com/spanner/docs/schema-and-data-model#create_a_parent_table) use PascalCase (instead of camelCase) for column names.
It'd be ideal if there was a convention set by Google that we follow here.